tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
Well, now I think you're just being difficult. The lines-of-code size of the *checked-in code* for test87.ans *does* matter to me (and also it would be nice to focus that test on just the new feature being added, instead of having the vast majority of it devoted to testing `HASH_DEL`). The size of the *generated* code is what I'm saying doesn't matter.